packages/grafana-prometheus/src/querycache/QueryCache.ts

```diff
@@ -268,7 +176,7 @@ export class QueryCache<T extends SupportedQueryTypes> {
       let targIdent = `${request.dashboardUID}|${request.panelId}|${targ.refId}`;
       let targSig = this.getTargetSignature(request, targ); // ${request.maxDataPoints} ?
 
-      if (this.shouldProfile && this.getProfileData) {
+      if (this.getProfileData) {
```

Comment:
@gtk-grafana Should I also remove this part? `pendingRequestIdsToTargSigs` seems doing nothing, doesn't it?

Reply:
Yeah, I think you can get rid of the `getProfileData` function as well. https://github.com/grafana/grafana/pull/65244/files, https://github.com/grafana/grafana/pull/70701/files

Reply:
More removal https://github.com/grafana/grafana/pull/89463/commits/29174512cc5b64ff4218c2c4ab8be79971184f72